test/exec_test.exs

```diff
@@ -237,6 +237,39 @@ defmodule QueryParser.ExecTest do
              ] = parse_and_exec(data(), %{"_id" => %{"$in" => [1, 2, 5]}})
     end
 
+    test "should return objet if find in array" do
+      assert [
+               %{"_id" => 1}
+             ] = parse_and_exec(data(), %{"ids" => 1})
+    end
+
+    test "should return objet if find in array with function" do
+      assert [
+               %{"_id" => 1}
+             ] = parse_and_exec(data(), %{"ids" => %{"$lt" => 2}})
+    end
+
+    test "should return objet if find in array with many function" do
+      assert [
+               %{"_id" => 7},
+               %{"_id" => 8},
+               %{"_id" => 9},
+               %{"_id" => 10}
+             ] = parse_and_exec(data(), %{"prev" => %{"$lt" => 2, "$gt" => 5}})
```

Comment:
```suggestion
             ] = parse_and_exec(data(), %{"prev" => %{"$lt" => 2, "$gt" => 5}})
```

How is it possible to obtain values that are less than 2 and greather than 5 at the same time ?